sql/moz-fx-data-shared-prod/search_terms_derived/unsanitized_search_term_sample_data/metadata.yaml

```diff
@@ -0,0 +1,26 @@
+friendly_name: Unanitized Firefox Search Terms Sample for Data Validation
+owners:
+  - ctroy@mozilla.com
+labels:
+  incremental: true
+description: |-
+  A 1% sample of raw search terms submitted to Firefox.
+
+  This table allows for data validation on raw data in the event
+  that the search volume metrics change in concerning ways.
+
+  See https://github.com/mozilla/search-terms-sanitization
+  for the code that populates this table.
+bigquery:
+  time_partitioning:
+    field: timestamp
+    type: day
+    require_partition_filter: true
+    expiration_days: 15
+workgroup_access:
+  - role: roles/bigquery.dataViewer
+    members:
```

Comment:
I don't think this is the appropriate classification for the data given https://docs.google.com/document/d/1TXMNjSRUsYgBTyP1HsQz8MkOqTY6FgfwPLa6JOeyXmg/edit#heading=h.834bdis4eiun.